<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProductForm</class>
 <widget class="QWidget" name="AddProductForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>170</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>230</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_buy">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>290</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sell">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>350</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_stock">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>410</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>نام کالا</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_code">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>کد کالا</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_buy">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>296</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>قیمت خرید</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sell">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>قیمت فروش</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stock">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>420</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>تعداد</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addProduct">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>480</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ثبت</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_menu">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>560</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>منو</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
